embedded/sql/engine.go

```diff
@@ -40,7 +40,10 @@ var ErrTableDoesNotExist = errors.New("table does not exist")
 var ErrColumnDoesNotExist = errors.New("column does not exist")
 var ErrColumnNotIndexed = errors.New("column is not indexed")
 var ErrLimitedKeyType = errors.New("indexed key of invalid type. Supported types are: INTEGER, VARCHAR[256] OR BLOB[256]")
+var ErrAutoIncrementWrongType = errors.New("auto incremented column need to be INTEGER type")
+var ErrAutoIncrementMultiple = errors.New("several auto incremental column were found. Wrong schema")
 var ErrLimitedAutoIncrement = errors.New("only INTEGER single-column primary keys can be set as auto incremental")
+var ErrLimitedUpsert = errors.New("upsert is only supported in tables without secondary indexes")
```

Comment:
rare... this limitation was already overcomed in master branch

Reply:
oh my bad I think I re-implemented that when merging with master branch locally

Reply:
with that we also could remove the `ErrLimitedAutoIncrement` error because it has been detailed `ErrAutoIncrementWrongType` (and all other cases do not generate any errors now)